crates/bins/src/bin/datadog-static-analyzer.rs

```diff
@@ -55,10 +55,20 @@ fn print_configuration(configuration: &CliConfiguration) {
     let ignore_paths_str = if configuration.path_config.ignore.is_empty() {
         "no ignore path".to_string()
     } else {
-        configuration.path_config.ignore.join(",")
+        configuration
+            .path_config
+            .ignore
+            .iter()
+            .map(|p| p.clone().into())
+            .collect::<Vec<String>>()
+            .join(",")
     };
     let only_paths_str = match &configuration.path_config.only {
-        Some(x) => x.join(","),
+        Some(x) => x
+            .iter()
+            .map(|p| p.clone().into())
+            .collect::<Vec<String>>()
+            .join(","),
```

Comment:
Curious why this change? Seems more complicated to read?

Reply:
Because that was the way I knew how to coerce a vector of PathPattern into a vector of String. Now I used some Rust 🪄 to solve it.